From 7ba5da7e4738058571664ac14312327bdab7c86d Mon Sep 17 00:00:00 2001
From: Peter Yin <peter.yin@quantatw.com>
Date: Fri, 9 Aug 2024 15:21:47 +0800
Subject: [PATCH 1018/1019] ARM: dts: aspeed: Harma: add rtc device

Add "nxp,pcf8563" device and the slave address is 0x51.

lore:https://lore.kernel.org/all/20240820102904.1756785-2-peteryin.openbmc@gmail.com/

Signed-off-by: Peter Yin <peter.yin@quantatw.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
index 8b0f10db03b0..b185badcde2a 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
@@ -137,10 +137,6 @@ &mac3 {
 	ncsi-package = <1>;
 };
 
-&rtc {
-	status = "okay";
-};
-
 &fmc {
 	status = "okay";
 
@@ -528,6 +524,11 @@ imux31: i2c@3 {
 			#address-cells = <1>;
 			#size-cells = <0>;
 			reg = <3>;
+
+			rtc@51 {
+				compatible = "nxp,pcf8563";
+				reg = <0x51>;
+			};
 		};
 	};
 };
-- 
2.25.1

